ImageFeed/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  ProfileViewController.swift
+//  ImageFeed
+//
+
+import UIKit
+
+final class ProfileViewController: UIViewController {
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        .lightContent
+    }
+
+    private enum Constants {
```

Comment:
Интересное решение, но я бы все-таки порекомендовал использовать структуру, а не enum. Но enum тоже можно использовать, но лучше не со статиками, а с case - будет использоваться сила enum ) 
private enum Constants: CGFloat {
       case someCase = 15.0
}

тогда someCase будет возвращать 15.0 при таком запросе: Constants.someCase.rawValue

Reply:
Считаю, что решение со `static` в `enum` имеет место быть и хорошо тем, что не будет риска инициализировать структуру, когда не надо. В [swiftlint](https://realm.github.io/SwiftLint/convenience_type.html) такое правило даже есть - `Types used for hosting only static members should be implemented as a caseless enum to avoid instantiation`